src/main/java/woowacourse/auth/config/AuthenticationPrincipalConfig.java

```diff
@@ -22,12 +22,16 @@ public AuthenticationPrincipalConfig(AuthService authService, JwtTokenProvider j
     @Override
     public void addInterceptors(InterceptorRegistry registry) {
         registry.addInterceptor(authCheckInterceptor())
-                .addPathPatterns("/customers/me");
+                .addPathPatterns("/customers/me")
+                .addPathPatterns("/customers/me/carts")
+                .addPathPatterns("/customers/orders")
+                .excludePathPatterns("/*");
+
```

Comment:
```suggestion
                .addPathPatterns("/**");
                .excludePathPatterns(...)
                .excludePathPatterns(...)
```
피드백 드린 방향은 이런 형태였습니다!